packages/astro/src/default/components/Header.astro

```diff
@@ -1,15 +1,30 @@
 ---
 import fs from 'node:fs';
+import path from 'node:path';
 import { LoginButton } from './LoginButton';
 import { ThemeSwitch } from './ThemeSwitch';
 import { useAuth } from './setup';
 
+const logoFilenames = [
+  'logo.svg',
+  'logo.png',
+  'logo.jpeg',
+  'logo.jpg',
+];
+
 let logo;
 
-try {
-  logo = fs.readFileSync('./public/logo.svg', { encoding: 'utf8' });
-} catch {
-  console.warn('No logo.svg found in public/');
+for (const logoFilename of logoFilenames) {
+  const exists = fs.existsSync(path.join('public', logoFilename));
+
+  if (exists) {
+    logo = `/${logoFilename}`;
+    break;
+  }
+}
+
+if (!logo) {
+  console.warn(`No logo found in public/. Support filenames: ${logoFilenames.map((filename) => `'${filename}'`).join(', ')}`);
```

Comment:
```suggestion
  console.warn(`No logo found in public/. Supported file names are: ${logoFilenames.map((filename) => `'${filename}'`).join(', ')}`);
```

Reply:
Instead of repeating the entire file name with every extension, what if we printed something like `logo.(svg|png|jpeg|jpg)` or something akin so it's a bit less repetitive? At least if the actual name is always the same.

Reply:
Oh very good point! Will update the PR with your suggestion 👍

Reply:
Updated it.

Reply:
Nice 🔥